### PR TITLE
Bug 1735548: pipeline_metadata.collector.ipaddr6 is incorrect when using Fluentd as log collector.

### PR DESCRIPTION
**Pull request by VladMasarik**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1735548 .
Part of that Bugzilla is also Rsyslog fix for the same incorrect field. However, the fix requires opening a PR in Cluster Logging Operator repository.

This is the Rsyslog PR https://github.com/openshift/cluster-logging-operator/pull/225

Additionally I have executed the `hack/update-fluentd-vendor-gems.sh` script, and committed changes it done.

### CODE REVIEW

File: fluentd/run.sh [hunk 1]
```diff
@@ -70,6 +70,9 @@ fi
 
 IPADDR4=`/usr/sbin/ip -4 addr show dev eth0 | grep inet | sed -e "s/[ \t]*inet \([0-9.]*\).*/\1/"`
 IPADDR6=`/usr/sbin/ip -6 addr show dev eth0 | grep inet6 | sed "s/[ \t]*inet6 \([a-f0-9:]*\).*/\1/"`
+
+IPADDR6="" # So as to omit this field from logs. See https://github.com/openshift/cluster-logging-operator/pull/225
```
Comment (VladMasarik) [inline fluentd/run.sh]: As you can see WIP as well, and I have not tested it yet.

File: fluentd/run.sh [hunk 2]
```diff
@@ -70,6 +70,9 @@ fi
 
 IPADDR4=`/usr/sbin/ip -4 addr show dev eth0 | grep inet | sed -e "s/[ \t]*inet \([0-9.]*\).*/\1/"`
```
Comment (richm) [inline fluentd/run.sh]: change this to 
```bash
IPADDR4=${NODE_IPV4:-$( /usr/sbin/ip -4 addr show dev eth0 | grep inet | sed -e "s/[ \t]*inet \([0-9.]*\).*/\1/" )}
```

File: fluentd/run.sh [hunk 3]
```diff
@@ -70,6 +70,9 @@ fi
...
 IPADDR6=`/usr/sbin/ip -6 addr show dev eth0 | grep inet6 | sed "s/[ \t]*inet6 \([a-f0-9:]*\).*/\1/"`
```
Comment (richm) [inline fluentd/run.sh]: comment out this line, or just remove it

File: fluentd/vendored_gem_src/fluent-plugin-viaq_data_model/lib/fluent/plugin/filter_viaq_data_model.rb [hunk 4]
```diff
@@ -203,7 +203,12 @@ def configure(conf)
         @docker_hostname = nil
```
Comment (richm) [inline fluentd/vendored_gem_src/fluent-plugin-viaq_data_model/lib/fluent/plugin/filter_viaq_data_model.rb]: This change needs to be proposed as a PR to https://github.com/ViaQ/fluent-plugin-viaq_data_model - then we will pull it into fluentd from there

**Comment by richm** [comment]
https://github.com/ViaQ/fluent-plugin-viaq_data_model/releases/tag/0.0.20

Next step - run `hack/update-fluentd-vendor-gems.sh` to pull in the new source (which will also pull in any other updates)

**Pull request merged**